pyrasterframes/src/main/python/tests/__init__.py

```diff
@@ -94,3 +94,7 @@ def create_layer(self):
         self.rf = rf.withColumn('tile2', rf_convert_cell_type('tile', 'float32')) \
             .drop('tile') \
             .withColumnRenamed('tile2', 'tile').as_layer()
+
+        df = self.spark.read.raster(self.img_uri)
+        self.df = df.withColumn('tile', rf_convert_cell_type('proj_raster', 'float32')) \
```

Comment:
There are probably several places where we should deprecate `rf` in favor of `df` in the tests...

Reply:
Yes, good point. Going to put this off for now though.